src/DataSource/DB.php

```diff
@@ -21,10 +22,10 @@ public static function setProvider(DataProvider $provider): void
         static::$provider = $provider;
     }
 
-    public static function load(?string $tenantId, string $name, string $id): ?stdClass
+    public static function load(?string $tenantId, string $name, string $id, ?string $resultType = Result::DEFAULT): ?stdClass
     {
         StopWatch::start(__METHOD__);
-        $result = static::get()->load($tenantId, $name, $id);
+        $result = static::get()->load($tenantId, $id, $resultType);
```

Comment:
Warum fällt `$name` beim Aufruf weg bzw wird durch `$id` ersetzt, in der Definition aber nicht?